src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -195,6 +217,9 @@ var ReactDOMSelect = {
       );
       didWarnValueDefaultValue = true;
     }
+
+    warnIfDuplicateValues(inst);
```

Comment:
This should be placed within the `_DEV_` block at the top of the function so it only runs in development.